modules/infra/submodules/storage/variables.tf

```diff
@@ -122,3 +122,10 @@ variable "network_info" {
     })
   })
 }
+
+variable "flyte_enabled" {
```

Comment:
should be moved into the `storage` dict

Reply:
I put this outside of storage because eventually we will need to use `flyte_enabled` in other areas as well, but those other changes are still up in the air and out of scope for this initial PR. The intention was that making it a top level key would save us some refactoring effort down the line